download_videos_youtube.py

```diff
@@ -0,0 +1,39 @@
+'''
+Script to download videos from youtube given a
+text file containing youtube links (one link per line).
+'''
+
+from __future__ import unicode_literals
+import argparse
+import  copy
+import youtube_dl
+
+
+def parser():
+    # process input options
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--links_file",
+        type=str,
+        help="Youtube List of links to download",
+        required=True
+        )
+    return parser.parse_args()
+
+
+def main():
+    args = parser()
+    ydl_opts = {}
+    with youtube_dl.YoutubeDL(ydl_opts) as ydl:
```

Comment:
If ydl_opts is an empty dict, why not `with youtube_dl.YoutubeDL({}) as ydl:`

Reply:
Because if someone wants to add some ytdl options it will be easy to identify.